Detections/CarbonBlack/KnownMalwareDetected.yaml

```diff
@@ -4,7 +4,7 @@ description: |
   'This creates an incident when a known Malware is detected on a endpoint managed by a Carbon Black.'
 severity: Medium
 requiredDataConnectors:
-  - connectorId: VMwareCarbonBlack
+  - connectorId: CustomConnector
```

Comment:
Please remove this, this is not custom.  We have a connector in the GitHub that is moving into the UX - https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/VMware%20Carbon%20Black/VMwareCarbonBlack_API_FunctionApp.json

Reply:
reverted change.